lib/vsc/install/shared_setup.py

```diff
@@ -1376,17 +1403,23 @@ def parse_target(self, target, urltemplate=None):
             if 'github' in urltemplate:
                 new_target['download_url'] = "%s/tarball/master" % new_target['url']
 
-        # Readme are required
-        readme = os.path.join(self.REPO_BASE_DIR, README)
-        if not os.path.exists(readme):
-            raise Exception('README is missing (was looking for %s)' % readme)
+        # README is required, use description file from target and remove it
```

Comment:
the `is required` comment is only relevant where the the `locate_readme` is called.

Reply:
it has been a while, but how could the `description_file` be missing?

Reply:
I moved that part of the comment. On the other hand, since `decription_file` is set in a different method, the fallbacks just cover the unlikely case that `parse_target` is called without going through `action_target`.